code/modules/games/cah.dm

```diff
@@ -0,0 +1,262 @@
+	"Scientists are not allowed to make Gatling _____.",
+	"It's not a party until the ____ arrive.",
+	"No matter how many Tajara you have, _____ is never acceptable.",
+	"No, your robot's first law is NOT to serve _____.",
+	"The robots are not disposal bins for your _____.",
+	"You can never have too many _____ on board.",
+	"Failing to fit ______ up the _____ . (Pick 2)",
+	"_____ has no idea what S&M is and _____ shows them. (Pick 2)",
+	"What's this in my hand?",
+	"My lover left me for ______.",
+	"The _____ apocalypse comes.",
+	"Hi! I'm Malyr Krissen, and I got hit by ______.",
+	"Ah. ______ . Good times.",
+	"Well, well. If it isn't _____ .",
+	"Please welcome our newest recruit: _____ !",
+	"Didn’t you hear? Kazimir Boytsov just got sick from _____ !",
+	"\"Tau Ceti has become a beacon of liberty and hope for ______ .\"",
+	"\"This colony of the Alliance has been proliferating _____ as potential weapons of mass destruction.\"",
+	"What is the president's next speech going to be about?",
+	"It's _____ time!",
+	"That was a wild party last night. I only remember _____ .",
+	"Create a haiku. (Draw 2; Pick 3)",
+	"You can't go on like this. Go talk to the doctor about _____ .",
+	"And the award goes to...",
+	"Look! Over there! It's _____ !",
+	"What you call a passion fruit, I call a _____ .",
+	"How do I deal with my grief?",
+	"Knock knock! Who's there?",
+	"Drats! Foiled by _____ AGAIN!"
+	)
+
+// White cards.
+/obj/item/deck/cah/var/list/card_text_list = list(
+	"Those motherfucking carp",
+	"Having sex in a sewer outlet",
+	"Space 'Nam",
+	"Space cowboys",
+	"My gardener getting SUPER high",
+	"Your boss thinking they're a badass",
+	"Being in a religious cult",
+	"Racially-biased directives",
+	"An Unathi who WON'T STOP FIGHTING",
+	"Tajaran fetishists",
+	"Bald dogs",
+	"Mummy-dear's sweet whispers",
+	"Being sucked out into space",
+	"Touchy-feely shells",
+	"Construction (again)",
+	"Tsunami",
+	"The ClockWorks on a Tuesday night",
+	"Fucking synths",
+	"Man-eating purple pod plants",
+	"Filling fire extinguishers with hydrogen",
+	"Librarians",
+	"Squids",
+	"Cats",
+	"Lizards",
+	"Apes",
+	"Trees",
+	"Ants",
+	"Supermatter undergarments",
+	"Bluespace",
+	"Vending machine tech singularity",
+	"Kitchen utensil anomalies",
+	"Locked lockers",
+	"Energy spatulas",
+	"Flashbang shoes",
+	"Space whales",
+	"Space sharks",
+	"Augmented male enhancers",
+	"Gyrating barks",
+	"Forehead-mounted laser weaponry",
+	"Pelvis-mounted laser weaponry",
+	"Cardboard cutouts",
+	"An obscene amount of rubber ducks",
+	"Brain cakes",
+	"A drone's game of life.",
+	"Anomaly suits",
+	"A hydrofarmer smoking reishi",
+	"Military-grade baseball bats",
+	"Barricading your classroom",
+	"An irritatingly chipper robot",
+	"Androids hanging out at a bar drinking beer",
+	"Flannel lingerie",
+	"A seventeen-year-old medical doctor",
+	"The throbbing erection that the police chief gets at the thought of shooting something",
```

Comment:
What police chief.=?

Reply:
Cards Against the Galaxy is localised according to the area it would be retailed in, I would imagine, in order for it to make a bit more sense. In this instance, I would've presumed a police chief would exist somewhere who wouldn't necessarily be a specific person but rather one that its audience can denote.
However, I do see potential consequences with this for character mentalities I don't want to help introduce into the server if it conflicts with the lore, especially if they would purely be out of real-life prerogative—I'm aware there had been several riots in the lore, but do you think there would be any reason for this to be in the card game? I myself have already had some doubts, but I wish to hear from you.